kokoro/linux/bazel/continuous.cfg

```diff
@@ -1,16 +1 @@
-# Config file for running tests in Kokoro
-
-# Location of the build script in repository
-build_file: "protobuf/kokoro/linux/bazel.sh"
-timeout_mins: 15
-
-env_vars {
-  key: "BAZEL_TARGETS"
-  value: "//src/..."
-}
-
-action {
-  define_artifacts {
-    regex: "**/sponge_log.*"
-  }
-}
+# Keep this empty!
```

Comment:
I like it better when it says "Use common.cfg instead'

Reply:
Whoops, you managed to find the one file I was using as a POC and got missed by my find/replace :P